executor/distsql.go

```diff
@@ -118,11 +118,20 @@ func closeAll(objs ...Closeable) error {
 	return errors.Trace(err)
 }
 
-// timeZoneOffset returns the local time zone offset in seconds.
-func timeZoneOffset(ctx sessionctx.Context) int64 {
-	loc := ctx.GetSessionVars().GetTimeZone()
+// zone returns the current timezone name and timezone offset in seconds.
+// In compatible with MySQL, we change `Local` to `System`.
+// TODO: Golang team plan to return system timezone name intead of
+// returning `Local` when `loc` is `time.Local`. We need keep an eye on this.
+func zone(ctx sessionctx.Context) (string, int64) {
```

Comment:
first DONE. 

For the second one, `time` package having the same `Zone` method does the same thing. So I prefet `zone` rather than `getTimeZone`. What do you think?

Reply:
When we call `time.Zone()`, we know this function returns a time zone, because the package name can give us some info to help us understand the functionality. But this function is in the `executor` package, its looks wired if the function just named `zone`. Maybe we can rename it to `timeZone` to match the function `time.Zone`?